packages/dds/shared-object-base/src/sharedObject.ts

```diff
@@ -36,6 +40,9 @@ export abstract class SharedObjectCore<TEvent extends ISharedObjectEvents = ISha
     extends EventEmitterWithErrorHandling<TEvent> implements ISharedObject<TEvent> {
     public get IFluidLoadable() { return this; }
 
+    private static readonly DdsOpProcessingTelemetrySymbol: symbol = Symbol("DdsOpProcessingTelemetrySymbol");
```

Comment:
also. declaring this as a non-exported const variable outside the class will be cheaper, and exposed less

Reply:
I thought of another issue with the singleton design. we can't just track per dds, it needs to be per instance as there can be multiple different usages of a single dds in an application, and we don't want to bundle the reporting. in logging we basically want the data per sharedObjectId which is a default property of all dds logs

Reply:
Technically it could still be done with the singleton by having the symbol be an instance-level variable instead of static. But I'll replace the singleton based on the rest of the feedback, so it'll be a moot point :).